crates/dojo-test-utils/src/anvil.rs

```diff
@@ -0,0 +1,45 @@
+use std::time::Duration;
+
+use ethers::middleware::SignerMiddleware;
+use ethers::providers::{Http, Provider};
+use ethers::signers::{LocalWallet, Signer};
+use ethers::utils::AnvilInstance;
+use ethers_core::utils::Anvil;
+
+pub struct TestAnvil {
+    url: String,
+    pub anvil: AnvilInstance,
+}
+
+impl TestAnvil {
+    pub async fn start() -> Self {
+        let port = 8545u16;
```

Comment:
If we use a fix port, we may not be able to have several tests running at the same time. We should here do one the following:
1. If we can let the OS choose the port, it's the best part.
2. Generate a random port to limit chances of conflicts. For that we should use high ports number (above 25_000 for instance).